XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/CameraSystem/BaseCameraDataProvider.cs

```diff
@@ -313,6 +310,21 @@ public override void Destroy()
 
         #endregion IMixedRealitySerivce Implementation
 
+        private void EnsureCameraRigSetup()
+        {
+            if (CameraRig == null)
+            {
+                if (CameraCache.Main.transform.parent.IsNull())
+                {
+                    var rigTransform = new GameObject().transform;
```

Comment:
Probably a good idea to give this object a name

Reply:
I hadn't changed that code, but sure, we can give it the same default as in other places.

Reply:
Added toolkit default for the default camera rig name and applied it to all places referring to or creating the default camera rig